andela/reaction_commerce_reactors
# #145751639 Make the API endpoints RESTful

### What does this PR do?
Makes API endpoints restful

 ### Description of Task to be completed?
Using **meteor-restivus** to set up CRUD endpoints. Extend Meteor DDP methods and provide restful API

### How should this be manually tested?
Using postman, check to ensure HTTP requests work for every Item collection as specified in files as  well as for the users

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2030083/stories/145751639

### Screenshots

<img width="759" alt="screen shot 2017-05-30 at 11 25 37 am" src="https://cloud.githubusercontent.com/assets/27012508/26574742/29f06eb4-452b-11e7-9d33-d1ba48413019.png">
<img width="1280" alt="screen shot 2017-05-30 at 11 25 20 am" src="https://cloud.githubusercontent.com/assets/27012508/26574746/2d3776f8-452b-11e7-9206-99cf5a179c76.png">
